ApiReview/API.Polly.Core/NoDocs/Polly.CircuitBreaker/BrokenCircuitException.TResult.cs

```diff
@@ -0,0 +1,14 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Polly.CircuitBreaker;
+
+public class BrokenCircuitException<TResult> : BrokenCircuitException
+{
+    public TResult Result { get; }
```

Comment:
what is the role of this result? if a CB is open, there is no result, right?

Reply:
It returns result that opened the circuit. It's V7 API and we need to keep it.

Reply:
Let's move this back to Polly and v8 will only throw `BrokenCircuitException`